index.js

```diff
@@ -172,28 +172,29 @@ ethjsPersonalSignButton.addEventListener('click', function(event) {
   var params = [from, msg]
 
   // Now with Eth.js
-  var eth = new Eth(web3.currentProvider)
+  const provider = await detectEthereumProvider();
+  var eth = new Eth(provider)
 
   eth.personal_sign(msg, from)
-  .then((signed) => {
-    console.log('Signed!  Result is: ', signed)
-    console.log('Recovering...')
+    .then((signed) => {
+      console.log('Signed!  Result is: ', signed)
+      console.log('Recovering...')
 
-    return eth.personal_ecRecover(msg, signed)
-  })
-  .then((recovered) => {
+      return eth.personal_ecRecover(msg, signed)
+    })
+    .then((recovered) => {
 
-    if (recovered === from) {
-      console.log('Ethjs recovered the message signer!')
-    } else {
-      console.log('Ethjs failed to recover the message signer!')
-      console.dir({ recovered })
-    }
-  })
+      if (recovered === from) {
+        console.log('Ethjs recovered the message signer!')
+      } else {
+        console.log('Ethjs failed to recover the message signer!')
+        console.dir({ recovered })
+      }
+    })
```

Comment:
A lot of whitespace is being introduced that looks uneeded? Is there a lint rule this is enforcing?

Reply:
It might be my local IDE. I'll fix it 🙂